content/blog/2020-12-14-december-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,171 @@
+  for the core DVC team- someone with strong Python development skills who can
+  build and ship essential DVC features.
+
+- A
+  [**Developer Advocate**](https://weworkremotely.com/remote-jobs/iterative-developer-advocate)
+  to support and inspire developers by creating new content like blogs,
+  tutorials, and videos- plus lead outreach through meetups and conferences.
+
+Does this sound like you or someone you know? Be in touch!
+
+### Video docs complete!
+
+As you may have heard
+[last month](https://dvc.org/blog/november-20-dvc-heartbeat), we've been working
+on adding complete video docs to the "Getting Started" section of the DVC site.
+We now have 100% coverage! We have videos that mirror the tutorials for:
+
+- [Data versioning](https://dvc.org/doc/start/data-versioning) - how to use Git
+  and DVC together to track different versions of a dataset
+
+- [Data access](https://dvc.org/doc/start/data-access) - how to share models and
+  datasets across projects and environments
+
+- [Pipelines](https://dvc.org/doc/start/data-pipelines) - how to create
+  reproducible pipelines to transform datasets to features to models
+
+- [Experiments](https://dvc.org/doc/start/experiments) - how to do a `git diff`
+  for models that compares and visualizes metrics
+
+https://media.giphy.com/media/L4ZZNbDpOCfiX8uYSd/giphy.gif
+
+The
+[full playlist is on our YouTube channel](https://www.youtube.com/playlist?list=PL7WG7YrwYcnDb0qdPl9-KEStsL-3oaEjg)-
+where, by the way, we've recently passed 2,000 subscribers! Thanks so much for
+your support. There's much more coming up soon.
+
+### Collaboration with GitLab
+
+We recently released a new blog with GitLab all about using [CML](cml.dev) with
+GitLab CI.
+
+https://twitter.com/gitlab/status/1334631001956487171
+
+You may notice that the tweet spelled our name differently, and since Twitter
+doesn't have an edit button, I think that means we're "Interative" now.
+[Hurry up and get your merch!](https://www.zazzle.com/t_shirt-235920696568133954)
+
+![](/uploads/images/2020-12-14/newname.png)
+
+### Workshops
+
+We gave a workshop at a virtual meetup held by the Toronto Machine Learning
+Society, and you can catch a video recording if you missed it. This workshop was
+all about getting started with GitHub Actions and CML!
+
+https://youtu.be/51H13lfHdMw
+
+DVC will also be represented at the upcoming
+[PyData Montreal](https://www.meetup.com/PyData-MTL/events/274786470/) online
+meeting organized by [Maria Khalusova](https://twitter.com/mariaKhalusova) and
+[Alexander Kim](https://twitter.com/alex000kim?lang=en). Myself (Elle) and
+[Josh Tobin](https://twitter.com/josh_tobin_) will be talking about approaches
+to full-stack machine learning. The event is free and virtual, so consider
+stopping by!
+
+![](/uploads/images/2020-12-14/pydata_mtl.jpg)
+
+## From the community
+
+There's no shortage of cool things to report from the community:
+
+### The DVC Udemy Course
```

Comment:
Mikhail had a special singup link for the singup - let's ask him and put it. It's important for him as I know.

Reply:
yes, good idea- asked on slack